### PR TITLE
Move continuation token vespa search tests to separate file

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

When working on PDCT-1274, I found that the search test files are huge and kinda grim to maintain. This PR splits the vespa search continuation token tests into a separate file as part five of a refactor to make the search tests more maintainable.

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [x] Skip auto-tagging
- [ ] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] GitHub workflow update
- [ ] Documentation update
- [ ] Refactor legacy code
- [ ] Dependency update

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] **DB_CLIENT DEPENDENCY IS ON THE LATEST VERSION**
- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

**Review by odrakes-cpr — APPROVED** [review]
lgtm

File: tests/search/vespa/test_vespa_search_cont_tokens.py
Comment (olaughter) [inline tests/search/vespa/test_vespa_search_cont_tokens.py:1]: does `pagination` express intent more clearly?

**Review by olaughter — APPROVED** [review]
Love to see it! One small suggestion below

File: tests/search/vespa/test_vespa_search_cont_tokens.py
Comment (katybaulch) [reply tests/search/vespa/test_vespa_search_cont_tokens.py:1]: Hmmm - yes in principle, but there are `pagination` tests and `continuation token` tests -- what's the distinction between these in the code?

Comment (olaughter) [reply tests/search/vespa/test_vespa_search_cont_tokens.py:1]: Reproducing from slack: 
> Vespa might find 1,000 results but returns 100 from the query. The backend then chunks this into smaller pieces of 10 results each with the option of getting another 100 from vespa

I would group these kinds of tests all together

Comment (katybaulch) [reply tests/search/vespa/test_vespa_search_cont_tokens.py:1]: Done :)

**Pull request merged**